docs/build_version_doc/setup_docker.sh

```diff
@@ -0,0 +1,17 @@
+# Setup Docker
```

Comment:
Docker is already installed on Ci, thus there's no need to have this script. I don't think that a script to do basic docker setup should be part of MXNet.

Reply:
Ok, I'm removing this.